### PR TITLE
Fixed licenses. Closes #428

### LINKED ISSUE #428: Missing licenses
It seems that the .rst files are all missing licenses, but it remains unclear whether they should have them in the first place. However, in addition to the `.rst` file, the following code files are missing licenses and should be updated:

@ommmid 
- [ ] creating_moveit_plugins/lerp_motion_planner/include/lerp_interface/lerp_planning_context.h
- [ ] trajopt_planner/src/trajopt_example.cpp

### PR DESCRIPTION
**Pull request by ommmid**
added the missing licenses, closes #428
@jliukkonen , please review

### CODE REVIEW

File: doc/creating_moveit_plugins/lerp_motion_planner/include/lerp_interface/lerp_planning_context.h [hunk 1]
```diff
@@ -1,8 +1,41 @@
...
+ *  COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT,
+ *  INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+ *  BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+ *  LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ *  CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+ *  LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+ *  ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+ *********************************************************************/
+
```
[the remark this answers is not in the document]
Comment (ommmid) [reply doc/creating_moveit_plugins/lerp_motion_planner/include/lerp_interface/lerp_planning_context.h]: done

File: doc/creating_moveit_plugins/lerp_motion_planner/src/lerp_example.cpp [hunk 2]
```diff
@@ -76,28 +76,28 @@ int main(int argc, char** argv)
   psm->startSceneMonitor();
   psm->startWorldGeometryMonitor();
   psm->startStateMonitor();
-  
+
   robot_model::RobotModelPtr robot_model = robot_model_loader->getModel();
 
-  // Create a RobotState and to keep track of the current robot pose and planning group                                                                 
+  // Create a RobotState and to keep track of the current robot pose and planning group
```
Comment (felixvd) [inline doc/creating_moveit_plugins/lerp_motion_planner/src/lerp_example.cpp:82]:
```suggestion
  // Create a RobotState to keep track of the current robot pose and planning group
```

**Review by felixvd — APPROVED** [review]
Friendly ping

**Comment by v4hn** [comment]
Superseded by #595

**Pull request closed without merging**